its/index.ts

```diff
@@ -0,0 +1,29 @@
+import fs from "fs";
+import path from "path";
+import { generateCombinations, serializeCombination } from "./combination";
+import { generatePipelineFile } from "./pipeline";
+
+const FIXTURES_PATH = path.join(__dirname, "fixtures");
+const FILENAME_PREFIX = "pipeline";
+
+export function generate() {
+  // Remove all .yml files from fixtures
+  const files = fs.readdirSync(FIXTURES_PATH);
+  for (const file of files) {
+    if (file.startsWith(FILENAME_PREFIX) && file.endsWith(".yml")) {
+      fs.unlinkSync(path.join(FIXTURES_PATH, file));
+    }
+  }
+
+  // Create new .yml files
+  for (const combination of generateCombinations()) {
+    const filename = `${FILENAME_PREFIX}-${serializeCombination(combination)}`;
+    const pipeline = generatePipelineFile(combination);
+
+    const pipelinePath = path.join(FIXTURES_PATH, `${filename}.yml`);
+    console.log(`Writing ${pipelinePath}`);
+    fs.writeFileSync(pipelinePath, pipeline);
+  }
+}
+
```

Comment:
I think if they are generated as part of the pipeline, we should ignore them from git

Reply:
As agreed, I'll drop them and try to find an automated solution right away

Reply:
(deleted my previous comment)
As discussed on Slack,  in order to be able to run them in AzDO, they need to be generated and available in the master branch